docs/src/components/Header/index.jsx

```diff
@@ -5,12 +5,16 @@ import { Navigate, OpenMenu, CloseMenu } from '../../actions'
 
 const OnSearch = (state, ev) => {
   ev.preventDefault()
-  return [Navigate, `/search?q=${encodeURI(ev.target.value)}`]
+  ev.target.search.select()
+  return [Navigate, `/search?q=${encodeURI(ev.target.search.value)}`]
 }
```

Comment:
This function produces several side effects, that should be wrapped in a proper effect.